src/aliceVision/localization/LocalizationResult.cpp

```diff
@@ -8,6 +8,10 @@
 #include <aliceVision/sfmDataIO/jsonIO.hpp>
 
 #include <boost/property_tree/ptree.hpp>
+
+// Avoids deprecation warning caused by internals of json_parser. This is properly fixed in
+// Boost 1.76.0: https://github.com/boostorg/property_tree/commit/d1c8825a45a0717e1ad79583d3283b0e5e32831e
```

Comment:
Perhaps, but people will likely compile AliceVision with older boost anyway. For example, the latest boost available on Debian and Ubuntu is 1.74.

Reply:
@fabiencastan I have simplified this PR now - instead of polluting the code, I added a line in CMakeLists.txt to set `BOOST_BIND_GLOBAL_PLACEHOLDERS`. This way the warning fix is much less intrusive.